pkg/runners/cairo_runner_test.go

```diff
@@ -156,3 +157,93 @@ func TestInitializeRunnerNoBuiltinsNoProofModeNonEmptyProgram(t *testing.T) {
 		t.Errorf("Wrong value for address 1:1: %d", rel)
 	}
 }
+
+func TestInitializeRunnerWithRangeCheckValid(t *testing.T) {
+	t.Helper()
+	// Create a Program with one fake instruction
+	program_data := make([]memory.MaybeRelocatable, 1)
+	program_data[0] = *memory.NewMaybeRelocatableFelt(lambdaworks.FeltFromUint64(1))
+	empty_identifiers := make(map[string]parser.Identifier, 0)
+	program_builtins := []string{builtins.CHECK_RANGE_BUILTIN_NAME}
+	program := vm.Program{Data: program_data, Identifiers: &empty_identifiers, Builtins: program_builtins}
+	// Create CairoRunner
+	runner, err := runners.NewCairoRunner(program)
+	if err != nil {
+		t.Errorf("NewCairoRunner error in test: %s", err)
+	}
+	// Initialize the runner
+	_, err = runner.Initialize()
+	if err != nil {
+		t.Errorf("Initialize error in test: %s", err)
+	}
+
+	builtin_runner := runner.Vm.BuiltinRunners[0]
+	if builtin_runner.Name() != builtins.CHECK_RANGE_BUILTIN_NAME {
+		t.Errorf("Name of runner builtin failed. Expected %s, got %s", builtin_runner.Name(), builtins.CHECK_RANGE_BUILTIN_NAME)
+	}
+
+	builtin_base := builtin_runner.Base()
+	expected_base := memory.NewRelocatable(2, 0)
+	if !builtin_base.IsEqual(&expected_base) {
+		t.Errorf("Base of runner builtin failed. Expected %d, got %d", expected_base, builtin_base)
```

Comment:
If the idea is to not capitalize the error strings, we should do it everywhere to keep consistency. For the scope of this PR, we can do it only for the lines added.